### PR TITLE
Vote 2023-09-12

### PR DESCRIPTION
**Pull request by BATMAH69**
### Description:
The **Omnibus vote** contain one motion:
1. Support **Jump Crypto voluntarily exits from the validator set** by setting `targetValidatorsCount` to 0. Jump Crypto requested it [here](https://research.lido.fi/t/lido-dao-proposal-to-set-targetvalidatorscount-for-jump-crypto-operator-to-0-to-wind-down-the-jump-crypto-legacy-set/5259) (items 1,2).
### Items:
  1) Grant STAKING_MODULE_MANAGE_ROLE to Lido Agent
  2) Set Jump Crypto targetValidatorsLimits to 0

### CODE REVIEW

File: tests/test_2023_09_12.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+    metadata = find_metadata_by_vote_id(vote_id)
+
+    assert get_lido_vote_cid_from_str(metadata) == "bafkreicca3ol5lhwcoq5fiaou2ikszspnev6jrfepwpiz5mvgo745ldd7m"
+    # 1)
+    assert staking_router.hasRole(STAKING_MODULE_MANAGE_ROLE, agent.address) == True
+
+    # 2)
+    NO_summary_after = nor.getNodeOperatorSummary(target_NO_id)
+    assert NO_summary_after[0]
```
Comment (TheDZhon) [inline tests/test_2023_09_12.py:77]:
```suggestion
    assert NO_summary_after[0] == True
```

let's be explicit maybe not idiomatic

File: tests/test_2023_09_12.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    accounts,
+):
+    ## parameters
+    agent = contracts.agent
+    nor = contracts.node_operators_registry
+    staking_router = contracts.staking_router
+    target_NO_id = 1
+    target_validators_count_change_request = TargetValidatorsCountChanged(
+        nodeOperatorId=target_NO_id, targetValidatorsCount=0
+    )
```
Comment (TheDZhon) [inline tests/test_2023_09_12.py:36]: I'd moved these definitions upper (at the global scope level)

File: utils/test/event_validators/allowed_recipient_registry.py
Comment (TheDZhon) [inline utils/test/event_validators/allowed_recipient_registry.py:1]: Why had this file been attached?

**Review by TheDZhon — COMMENTED** [review]
:eyes:

File: utils/test/event_validators/allowed_recipient_registry.py
Comment (BATMAH69) [reply utils/test/event_validators/allowed_recipient_registry.py:1]: This stay from first version of vote I will remove it

File: tests/test_2023_09_12.py (hunk 2 above)
Comment (BATMAH69) [reply tests/test_2023_09_12.py]: it doesn't in global scope, probably there some initialisation between global scope execute and test call
image

**Review by zuzueeka — APPROVED** [review]

**Review by bulbozaur — APPROVED** [review]
LGTM

File: tests/README.md [hunk 3]
```diff
@@ -61,3 +61,27 @@ the voting without modification of the common test files
...
+poetry run brownie console --network mainnet-fork
+```
+
+How to decode error messages?
+1) You need to clone `lido-cli` repo.
+```shell
+git clone https://github.com/lidofinance/lido-cli
+```
+2) install
+4) run
```
Comment (TheDZhon) [inline tests/README.md:84]: we missed 3) here?

P.S. Don't you need to copy and tune up the `.env` file before running?

**Review by TheDZhon — COMMENTED** [review]
:eyes:

File: tests/README.md (hunk 3 above)
Comment (BATMAH69) [reply tests/README.md]: fixed

**Pull request merged**